src/themes/teams/components/Icon/iconStyles.ts

```diff
@@ -86,11 +90,12 @@ const getIconSize = (size, sizeModifier): number => {
   return modifiedSizes[size] && modifiedSizes[size][sizeModifier]
 }
 
-const getIconColor = color => color || 'currentColor'
+const getIconColor = (colorProp: string, variables: IconVariables) =>
+  _.get(variables.colors, colorProp, variables.color || 'currentColor')
```

Comment:
note, this changes semantic of the method. Previously it was a feature that icon will take a color of its surrounding - with `color` variable being completely ignored. Please, ensure the following things before merging:
- corresponding changelog entry is provided in the BREAKING CHANGES section
- there are no visual regressions related to that

Reply:
> Previously it was a feature that icon will take a color of its surrounding - with color variable being completely ignored

@kuzhelov I am aware, but are you sure? from what I can see from previous implementation, I am not changing the logic:
```typescript
const getIconColor = color => color || 'currentColor'
```

where `color` param is `variables.color`.

It looks like priority was:
1. `variables.color`
2. `currentColor`

and after my change the priority is:
1. `color` prop
2. `variables.color`
3. `currentColor`

Reply:
this is the only change in screener, so no visual regressions:
![screenshot 2018-12-20 at 17 08 10](https://user-images.githubusercontent.com/5442794/50292492-e7eaf600-0479-11e9-9b2c-cb7f53d9843e.png)